src/gte.ts

```diff
@@ -0,0 +1,39 @@
+import curry from "./curry";
+import Curry from "./types/Curry";
+
+/**
+ * Returns true if the first argument is greater or equal than the second; false otherwise.
+ *
+ * @example
+ * ```ts
+ * gte(5, 1) // expected true
+ * gte(1, 1) // expected true
+ * gte(1, 5) // expected false
+ * gte("a", "b") // expected false
+ * gte("b", "a") // expected true
+ *
+ * filter(gte(5), [1, 2, 4, 5, 8, 9]) // Iterable<[5, 8, 9]>
+ * filter(gte(5), [1, 2, 3, 4]) // Iterable<[]>
+ * filter(gte("b"), ["a", "b", "c"]) // Iterable<["b", "c"]>
+ * filter(gte("b"), ["a"]) // Itreable<[]>
+ * ```
+ */
+function gte(a: string): Curry<(b: string) => boolean>;
```

Comment:
It should be able to accept the `Date` object as an argument as well.

Reply:
It's good idea, I will develop this

Reply:
but I have a question

compare date? or compare time?
The standards are ambiguous

Reply:
I mean just like this:

```ts
function gte(a: Date, b: Date) {
  return a >= b;
}

const d1 = new Date();
const d2 = new Date(d1);

gte(d1, d2); // true
```

Reply:
my question mean like this:
```ts
function gte(a : Date, b : Date){
  return a > b;
}

const d1 = new Date(2022, 4, 11);
const d2 = new Date(2022, 4, 11, 4, 10 ,10);

gte(d1, d2); // true? or false?
```

struggle between plan a that compare `a < b` and plan b that compare `a.getTime() < b.getTime()`

Maybe like this...
```ts
const compare_a = a instanceof Date ? a.getTime() : a;
return (b: string | number | Date): boolean => {
    if (typeof a !== typeof b) {
      throw new TypeError(
        "The values you want to compare must be of the same type",
      );
    }
    const compare_b = b instanceof Date ? b.getTime() : b;
    return compare_b < compare_a;
  };
```

I don't know the universal plan.

Reply:
There is no reason to use the `getTime` method. Just using the comparison operator is enough.

Reply:
Um... I think some user could use this to compare time.

But for now, I made it what you wanted.

Reply:
!.!
I realized this example is wrong
`gt(d1, d2); // true? or false?` 

I'm so sorry

thx.